### ISSUE TITLE
Teardown Script / Procedure

### ISSUE THREAD

**Issue by nparsons08** [segmentio/stack#9]
It would great if you provided a teardown procedure or script. As of now, we have to go through each resource and delete manually.

**Comment by calvinfo** [comment]
@nparsons08 does `terraform destroy` handle the case you're looking for? It should remove all of the resources you're tracking. 

We can certainly add it to the readme if so :)

**Comment by nparsons08** [comment]
Ah, yes. Just tried it and it worked perfectly. Would be great to see a small write up in the readme :) Thanks!

**Comment by gregwebs** [comment]
I don't think this requires documentation. Would be good to close out some issues.
